tests/SprykerTest/Glue/AppKernel/Controller/AppConfigureControllerTest.php

```diff
@@ -82,6 +82,31 @@ public function testPostConfigureReturnsSuccessResponseAndActivatesTheAppConfigu
         $this->tester->assertAppConfigIsActivated('tenant-identifier');
     }
 
+    /**
+     * @group single
+     *
+     * @return void
+     */
+    public function testPostConfigureReturnsSuccessResponseAndKeepsPreviousConfigurationWhenAnAppConfigurationExists(): void
+    {
+        // Arrange
+        $glueRequest = $this->tester->createGlueRequestFromFixture('valid-config-request');
+        $this->tester->havePersistedAppConfigTransfer([AppConfigTransfer::TENANT_IDENTIFIER => 'tenant-identifier', AppConfigTransfer::IS_ACTIVE => false, AppConfigTransfer::CONFIG => ['key' => 'value']]);
+        $appConfigController = $this->tester->createAppConfigController();
+
+        $expectedConfig = ['key' => 'value', 'clientId' => 'ClientID', 'clientSecret' => 'ClientSecret', 'isActive' => true];
+
+        // Act
+        $glueResponse = $appConfigController->postConfigureAction($glueRequest);
+
+        // Set the expected content for the validation
+        $glueRequest->setContent('{"data":{"type":"configuration","attributes":{"configuration":"{\"key\":\"value\",\"clientId\":\"ClientID\",\"clientSecret\":\"ClientSecret\",\"isActive\":true}"}}}');
```

Comment:
so you even have a test with `isActive` in the configuration form, but set it always to true.

Reply:
Same as for the other one. Let's do in a separate task.

Reply:
The configuration does not contain the isActive field from the main form. That would need to be added from the registry service and forwarded to the Apps. Then we can make use of it.

For now, we need to not delete the configuration when we disconnect as it would remove values that we can only get once. The Webhooksecret can only be gotten once when creating a Webhook. Since we need to keep the webhooks on deletion to continue receiving updates until the App has ended all its work we can't remove them.

When the registry-service forwards the isActive flag we can switch but also need to take into account to not delete the config when disconnected.

Reply:
Adding any configuration logic to the ATRS is "no go". All logic for an app has to be handled by an app. Anyway configuration.json is provided by an app.